gradle-conjure/src/main/java/com/palantir/gradle/conjure/CompileIrTask.java

```diff
@@ -97,6 +109,19 @@ public final File getExecutableDir() {
         return productDependencies;
     }
 
+    @Input
+    @Optional
+    public final MapProperty<String, Serializable> getConjureExtensions() {
+        return conjureExtensions;
+    }
+
+    @InputFile
+    @Optional
+    @PathSensitive(PathSensitivity.RELATIVE)
+    public final RegularFileProperty getExtensionsFile() {
```

Comment:
Do we need both the input file and property for now? Might be worth just using one for now to keep the API minimal but no strong opinion here.

Reply:
I think I'm okay with just the file for production, but the inline feels more complete and I have a hunch it may make some testing easier.  Haven't gotten all the way through testing with local version yet to know for sure.

Reply:
Sounds ok when it makes testing easier.